src/ImageSharp/Formats/Jpeg/Components/Decoder/YCbCrToRgbTables.cs

```diff
@@ -0,0 +1,95 @@
+﻿// <copyright file="YCbCrToRgbTables.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Formats.Jpg
+{
+    using System.Runtime.CompilerServices;
+
+    using ImageSharp.PixelFormats;
+
+    /// <summary>
+    /// Provides 8-bit lookup tables for converting from YCbCr to Rgb colorspace.
+    /// Methods to build the tables are based on libjpeg implementation.
+    /// </summary>
+    internal struct YCbCrToRgbTables
+    {
```

Comment:
@JimBobSquarePants see my comment on `RgbToYCbCrTables`, the same could be applied to `YCbCrToRgbTables`!

An additional thing: it's really important to move these 4 tables into a contigous memory area.  It will reduce the chance of cache misses. You have 2 options for this:
1. Do it like you did in `RgbToYCbCrTables`, and turn `YCbCrToRgbTables` into a static class
2. Turn all the four `static readonly int[]` arrays into `fixed` instance arrays.

I guess in case of `YCbCrToRgbTables` the second approach would be faster (no additions needed).